crates/install-wheel-rs/src/linker.rs

```diff
@@ -32,30 +31,13 @@ use crate::wheel::{
 /// Wheel 1.0: <https://www.python.org/dev/peps/pep-0427/>
 #[instrument(skip_all, fields(wheel = % wheel.as_ref().display()))]
 pub fn install_wheel(
-    location: &InstallLocation<impl AsRef<Path>>,
+    target: &Target,
```

Comment:
The client now passes in a struct that contains all the paths we need to use here (rather than passing in a root, and expecting this crate to construct paths).

Previously, this crate was responsible for constructing those paths, and it _assumed_ that it was installing into a virtual environment.